pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -7,12 +7,18 @@
 import dateutil
 import pandas as pd
 import pandas.util.testing as tm
-from pandas.compat import lrange
+from pandas.compat import lrange, PY3
 from pandas import (DatetimeIndex, Index, date_range, DataFrame,
                     Timestamp, offsets)
 
 from pandas.util.testing import assert_almost_equal
 
+if PY3:
+    from datetime import timezone
```

Comment:
this is not idiomatic, see comments above

Reply:
rather than put this here, make a fixture in pandas/conftest.py

called datetime_tz_utc to return this and skip if its not PY3